Lib/platform.py

```diff
@@ -725,6 +766,21 @@ def _get_machine_win32():
     # http://www.geocities.com/rick_lively/MANUALS/ENV/MSWIN/PROCESSI.HTM
 
     # WOW64 processes mask the native architecture
+    try:
+        arch, = _wmi_query('CPU', 'Architecture')
```

Comment:
```suggestion
        arch = _wmi_query('CPU', 'Architecture')[0]
```

Reply:
This also doesn't work because `_wmi_query` is a generator.

But I'll make it use `[arch, *_] =` on the LHS, which is more obvious.